source/user-guide/workflow/validation.rst

```diff
@@ -1,5 +1,31 @@
 .. _validation:
 
-Validation
-##########
+Validating records
+##################
 
+In the metadata editor, click the Validate button to trigger the validation.
+
+The right panel display all validation results by levels:
+
+.. figure:: img/validation.png
+
+Click thumbs up or down icon to display errors. Errors reported in blue are for information only and will not affect the global validation of the record:
+
+.. figure:: img/validation-details.png
+
+
+The editor reports also report error messages:
```

Comment:
The editor also reports error messages where applicable: (since not all error messages can be shown in the editor)